docker_util.py

```diff
@@ -0,0 +1,84 @@
+import json
+import subprocess
+
+from verbose_print import print_v
+
+
+class ImageNotFoundError(Exception):
+    def __init__(self, image):
+        self.image = image
+        self.message = f'The image {image} was not found.'
+
+    def __str__(self):
+        return self.message
+
+
+def tag_image(source, target, is_dry_run):
+    print(f'Tagging image {source} as {target}')
+    tag_command = ('docker', 'tag', source, target)
+    if is_dry_run:
+        print_v(' '.join(tag_command))
+        print('Image was not actually tagged since this is a dry run')
```

Comment:
a neat way to convey "dry run" might be to just `#` prefix the command when printing this

Reply:
Mite b cool
Added it